charts/nautobot/values.yaml

```diff
@@ -236,6 +235,23 @@ nautobot:
             value: 2
             periodSeconds: 15
 
+  # Add a Persistent Volume Claim for /static files of Nautobot
+  persistence:
```

Comment:
I think the name here should be more explicit, as we only create a persistent volume for static files. For example, `persistenceStaticFiles`.

Reply:
Agreed! Changed it